Proposed/asc049.md

```diff
@@ -2,21 +2,20 @@
 Add `concentration` Unit to `Units` Definitions
 
 #### **Authorship**
-Alex Hadik alex@transcriptic.com
+Alex Hadik alex.hadik@strateos.com
 
 #### **Motivation**
 We should formalize how we represent concentration so that we have a standard way of defining solution concentrations.
 
 #### **Proposal**
-Add an arbitrary `item` "unit" to the `units` definition list. This is used as a catch all for discrete items such as cells, beads, etc.
 
-Add three types of concentration units to the `units` definition list: mass, molarity and arbitrary item. These concentration types will be defined as follows:
+Add three types of concentration units to the `units` definition list: mass, volume and molarity. These concentration types will be defined as follows:
 
 - MassConcentration: `milligram/milliliter`, `nanogram/microliter`
+- VolumeConcentration: `milliliter/milliliter`, `microliter/microliter`
 - MolarConcentration: `mole/liter`, `millimole/liter`, `micromole/liter`
```

Comment:
As a separate note, it'll be good if we can support the `molar` syntax as well.

i.e. nanomolar, micromolar, millimolar, molar